minifi/pom.xml

```diff
@@ -0,0 +1,1100 @@
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi</artifactId>
+        <version>1.14.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.apache.nifi.minifi</groupId>
+    <artifactId>minifi</artifactId>
+    <version>1.14.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <description>Apache NiFi - MiNiFi is a child project effort to aid in the collection and transmission of data close to where it is created.</description>
+
+    <modules>
+        <module>minifi-commons</module>
+        <module>minifi-bootstrap</module>
+        <module>minifi-nar-bundles</module>
+        <module>minifi-docs</module>
+        <module>minifi-assembly</module>
+        <module>minifi-toolkit</module>
+        <module>minifi-docker</module>
+        <module>minifi-c2</module>
+        <module>minifi-integration-tests</module>
+    </modules>
+
+    <url>https://nifi.apache.org/minifi</url>
+    <organization>
+        <name>Apache NiFi Project - MiNiFi subproject</name>
+        <url>https://nifi.apache.org/</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+        </license>
+    </licenses>
+    <mailingLists>
+        <mailingList>
+            <name>Dev</name>
+            <subscribe>dev-subscribe@nifi.apache.org</subscribe>
+            <unsubscribe>dev-unsubscribe@nifi.apache.org</unsubscribe>
+            <post>dev@nifi.apache.org</post>
+            <archive>https://mail-archives.apache.org/mod_mbox/nifi-dev</archive>
+        </mailingList>
+        <mailingList>
+            <name>Users</name>
+            <subscribe>users-subscribe@nifi.apache.org</subscribe>
+            <unsubscribe>users-unsubscribe@nifi.apache.org</unsubscribe>
+            <post>users@nifi.apache.org</post>
+            <archive>https://mail-archives.apache.org/mod_mbox/nifi-users</archive>
+        </mailingList>
+        <mailingList>
+            <name>Commits</name>
+            <subscribe>commits-subscribe@nifi.apache.org</subscribe>
+            <unsubscribe>commits-unsubscribe@nifi.apache.org</unsubscribe>
+            <post>commits@nifi.apache.org</post>
+            <archive>https://mail-archives.apache.org/mod_mbox/nifi-commits</archive>
+        </mailingList>
+    </mailingLists>
+    <prerequisites>
+        <maven>${maven.min-version}</maven>
+    </prerequisites>
+    <scm>
+        <connection>scm:git:git://git.apache.org/nifi-minifi.git</connection>
+        <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/nifi-minifi.git</developerConnection>
+        <url>https://gitbox.apache.org/repos/asf?p=nifi-minifi.git</url>
+      <tag>HEAD</tag>
+    </scm>
```

Comment:
This can be done after merging this PR, but these repo URLs should be updated :)